DiscordBot/src/commands/djsCommand.ts

```diff
@@ -38,7 +38,23 @@ export default class DjsCommand extends BaseCommand {
             .then(res => res.json())
             .then(embed => {
                 if (embed && !embed.error) {
-                    commandData.message.channel.send({ embed });
+                    commandData.message.channel.send({ embed }).then(async m => {
+                        await (m as Message).react('🗑')
+                        const collector = (m as Message).createReactionCollector((reaction, user) => user.id === commandData.message.author.id, { time: 10000 });
+
+                        collector.on('collect', r => {
+                            if (r.emoji.name === '🗑') {
+                                r.message.delete();
+                                commandData.message.delete();
+                            }
+                        });
+
+                        collector.on('end', r => {
+                            if (r.size <= 0) {
+                                (m as Message).reactions.removeAll();
+                            }
+                        })
```

Comment:
;

Reply:
For this one, I am going to assume that it's suppose to go at the end of the `.then()` method with the Promise resolve from `TextChannel#send()`.

```js
commandData.message.channel.send(...).then(...)
```
Possibly here?